Automattic/jetpack
# tools: update package versioning for build action

#### Changes proposed in this Pull Request:

Start using `@dev` version numbers for packages in monorepo, which are then updated with stable versions numbers when built and pushed to the mirror repos.

#### Other information:


#### Jetpack product discussion

Internal: p9dueE-5YH-p2#comment-8775

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Simple PHP package example:

- Pull down this branch.
- Run `tools/check-intra-monorepo-deps.sh -vau` which will update package versions to `@dev`
- Then run `pnpm jetpack build -v --no-pnpm-install --for-mirrors="/tmp/jetpack-build" packages/connection`
- Make sure there are no errors output or build failures.
- Check that `/tmp/jetpack-build/Automattic/jetpack-connection/composer.json` has the correct expected version numbers (e.g., `^3.2`). The version number should match the most recent changelog version for that particular project.
